common/src/main/java/com/microsoft/identity/common/internal/ui/browser/BrowserDescriptor.java

```diff
@@ -83,10 +76,6 @@ public boolean matches(@NonNull Browser browser) {
             return false;
         }
 
-        if (mUseCustomTab != browser.isCustomTabsServiceSupported()) {
```

Comment:
should it be this instead?

if (mUseCustomTab){
    if (! browser.isCustomTabsServiceSupported()){
        return false;
    }
}

Reply:
I don't think we need to be specifying whether to use custom tabs in a browser or not. If the browser supports it, they will be used. If not, then they won't be used. I have therefore removed `mUseCustomTab` from the BrowserDescriptor class. Do you think we need to keep it there?